lang/lang-es.js

```diff
@@ -0,0 +1,591 @@
+(function (window) {
+/**
+ * {virtualclass1} {virtualclass2} are elements you passes with getString function eg:-
+ *   virtualclass.lang.getString('operaBrowserIssue', ['opeara', 27]);
+ *   opera and 27 will be replaced over the {virtualclass1} and {virtualclass2} resepectively
+ *   for particular line of language file.
+ * @type type
+ */
+    const message = {
+        ActiveAll: 'Activar todo',
+        addcontext: 'Añadir marcador',
+        Addoption: 'Añadir opción',
+        addnew: 'Añadir nuevo',
+        adminusr: 'Administrador',
+        appPrerequites: 'Validar requisitos del sistema',
+        assign: 'Asignar',
+        assignDisable: 'Recuperar controles',
+        askplayMessage: '<span id="askplaymsg"> ¿Deberíamos empezar la sesión?</span><br /><span id="remaingmsg">Los'
+        + ' datos restantes se descargarán en segundo plano.</span>',
+        askplaymsg: '<span id="askplaymsg"> "Descarga en curso, haga clic en Reproducir para comenzar </span>"',
+        audioInput: 'Entrada de audio',
+        audiolatency: 'Latencia de audio',
+        audioOutput: 'Salida de audio',
+        audioDisable: 'Dejar de silenciar',
+        audioEnable: 'Silenciar',
+        addQuestion: 'Añadir pregunta',
+        addContext: 'Añadir marcador',
+        answer: 'Respuestas',
+        action: 'Acciones',
+        audioTest: 'Su voz será grabada y reproducida para usted. \n Presione Ok y hable durante unos segundos.',
+        Back: 'Atrás',
+        vote: 'Votar',
+        bandfast: 'Conexión de red óptima',
+        bandmedium: 'Conexión de red lenta',
+        bandslow: 'Fallo en la conexión de red',
+        bandwitdhImageNotFound: 'Error al descargar la imagen, haga clic en el botón "Siguiente" para continuar.',
+        becomeTeacher: 'Convertirse en docente',
+        Binary_Limit_Exeed: 'Advertencia: Alto uso de BN',
+        bulkUserActions: 'Acciones masivas de usuario',
+        ByInstructor: 'Por instructor',
+        ByTimer: 'Por tiempo',
+        cancel: 'Cancelar',
+        canvasDrawMsg: 'Puede hacer clic en cualquier herramienta para dibujar' + 'haga clic y mantenga presionado'
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected string concatenation of literals.](https://app.codacy.com/gh/congrea/virtualclass/pullRequest?prid=5529949)